Engines/Wine/Plugins/application.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name"        : "Wine Plugins",
+    "id"          : "plugins",
```

Comment:
`plugins` or `wine_plugins`?

Reply:
`plugins` is enough. The whole include is `"engines", "wine", "plugins", ...`.